backend/src/users/users.service.ts

```diff
@@ -1,4 +1,167 @@
-import { Injectable } from '@nestjs/common';
+import {
+  BadRequestException,
+  Injectable,
+  InternalServerErrorException,
+  Search,
+} from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import {
+  User,
+  UserReservation,
+  VLending,
+  VLendingForSearchUser,
+} from 'src/entities';
+import { In, Like, Repository } from 'typeorm';
+import {
+  GetUserResponseDto,
+  GetUsersRequestDto,
+  GetUsersResponseDto,
+  UpdateUsersRequestDto,
+} from './dto/users.dto';
+import { getUserIncludes } from './users.enums';
+import { isStringInArrayCaseInsensitive } from 'src/common/utils/utils';
+import * as bcrypt from 'bcrypt';
 
 @Injectable()
-export class UsersService {}
+export class UsersService {
+  constructor(
+    @InjectRepository(User)
+    private readonly usersRepository: Repository<User>,
+    @InjectRepository(VLendingForSearchUser)
+    private readonly userLendingRepository: Repository<VLendingForSearchUser>,
```

Comment:
```suggestion
    private readonly vLendingForSearchUser: Repository<VLendingForSearchUser>,
```
어떤 repository 참조했는지 명확하게 써주세요